src/webui/api/appcontroller.h

```diff
@@ -39,6 +39,14 @@ class AppController : public APIController
 
 public:
     using APIController::APIController;
+    bool isActionSafe(const QString &actionName) const override
```

Comment:
Since this class now has `override`, you should try adding the `final` keyword to class:
`class AppController final : public APIController`
(This applies to every classes that has `override`)